### ISSUE TITLE
Add "date added" on actively hiring list?

### ISSUE THREAD

**Issue by LuffyWesley** [gcreddy42/hiring2020#221]

**Comment by gcreddy42** [comment]
Yeah. This is something I've been thinking about too. But I feel it'd serve the purpose and be minimalistic if we just add positions chronologically instead of alphabetically. What do you think?

**Comment by LuffyWesley** [comment]
True. Another thing that I was thinking about with the date, it makes it easier to check on postings that have been on the list for 2+ weeks.

**Comment by gcreddy42** [comment]
Agree. Let's add date then.
